sys-apps/onerng/onerng-3.6.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1 udev
+
+DESCRIPTION="Software for the Open Hardware Random Number Generator called OneRNG"
+HOMEPAGE="http://www.onerng.info/"
+SRC_URI="https://github.com/OneRNG/onerng.github.io/raw/master/sw/${PN}_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="app-crypt/gnupg
+	dev-python/python-gnupg[${PYTHON_USEDEP}]
+	sys-apps/rng-tools
+	sys-process/at
+	virtual/udev"
+
+DEPEND="virtual/pkgconfig
+	virtual/udev"
+
+S="${WORKDIR}/${PN}_${PV}"
+
+src_configure() {
+	# Run configure
+	econf
+}
+
+src_compile() {
+	# Run make
+	emake
+}
+
+src_install() {
+	# Install files
+	emake DESTDIR="${D}" install
```

Comment:
is not it default?

Reply:
Why do you mean?
As I did set DOCS for einstalldocs, I had to set also the emak command?
Or do you mean DESTDIR? Without it, make will fail, because it triggers the sandbox.